csharp/device/Microsoft.Azure.Devices.Client/HttpClientHelper.cs

```diff
@@ -123,7 +199,35 @@ sealed class HttpClientHelper : IHttpClientHelper
             return result;
 #endif
         }
+#endif
 
+#if NETMF
+        static HttpWebResponse ReadResponseMessageAsync(HttpWebResponse message)
+        {
+            // TODO
+            // unclear what is the purpose of this method and how to implement it
```

Comment:
Nothing to worry about. I latter figured this out but forgot to remove the method call.
In the original code it reads the HTTP response to get the ETag header and the message body (if any).
In NETMF it's more convenient to read the response stream at OnReceive and store the content in the new Message there.
As for the ETag it's read from the respective header at OnReceive too so we are not missing anything here.